vision/unit_tests/test_coordinate_lengths.py

```diff
@@ -0,0 +1,151 @@
+"""
+Testing vision.deskew.coordinate_lengths.py
+"""
+
+import unittest
+
+import numpy as np
+
+from vision.deskew.coordinate_lengths import latitude_length
+from vision.deskew.coordinate_lengths import longitude_length
+
+
+class TestCoordinates(unittest.TestCase):
+    """
+    Testing the coordinate lengths module
+    """
+
+    # Testing latitude_length
+
+    # Tests the maximum possible latitude in degrees as an input to the latitude_length function
+    def test_latitude_length_at_max_val(self) -> None:
```

Comment:
Another comment I would make is that your function names are kind of long. Once these two functions that you are testing are split into two test classes, this function for example could be renamed `test_max_val`